db/db_impl/db_impl.cc

```diff
@@ -71,6 +71,7 @@
 #include "options/cf_options.h"
 #include "options/options_helper.h"
 #include "options/options_parser.h"
+#include "port/jemalloc_helper.h"
```

Comment:
Should we put this `#include` also inside macro `#ifdef ROCKSDB_JEMALLOC`?

Reply:
I was debating it but not sure what the style is for this repo? I can make the change

Reply:
updated